controller/src/beerocks/master/db/db.h

```diff
@@ -107,6 +108,9 @@ class db {
         int blacklist_channel_remove_timeout;
         int failed_roaming_counter_threshold;
         int roaming_sticky_client_rssi_threshold;
+        int clients_persistent_db_max_size;
+        int max_timelife_delay_days;
+        int unfriendly_device_max_timelife_delay_days;
```

Comment:
its default value is 256 (it is not an index) so you probably meant `uint16_t` here also for the max clients size.

I used int since I saw the rest of the parameters in the master-config use `int` and thought it is intentional due to the fact it is read from configuration file and translated using `utils::stoi` and `string_utils::stoi`.

regarding the unfriendly:
not yet as it is not fully defined - this will be defined as part of the next phase of this task.

Reply:
On Veriview we check 255 clients, not 256.

Reply:
again - you refer to currently connected clients while this parameter defines the "cache" of clients we know about/configured a steering behavior about.
there can be 30 max clients connected simultaneously but many more defined in the db